tests/integration/helpers/element-test.js

```diff
@@ -264,4 +245,88 @@ module('Integration | Helper | element', function(hooks) {
 
     assert.dom('p').exists();
   });
+
+  module('invalid usages', function() {
+    test('it requires at least one argument', async function() {
+      expectEmberError(new Error('Assertion Failed: The `element` helper takes a single positional argument'));
+
+      await render(hbs`
+        <div>
+          {{#let (element) as |Tag|}}
+            <Tag id="content">hello world!</Tag>
+          {{/let}}
+        </div>
+      `);
+    });
+
+    test('it requires no more than one argument', async function() {
+      expectEmberError(new Error('Assertion Failed: The `element` helper takes a single positional argument'));
+
+      await render(hbs`
+        <div>
+          {{#let (element "h1" "h2") as |Tag|}}
+            <Tag id="content">hello world!</Tag>
+          {{/let}}
+        </div>
+      `);
+    });
+
+    test('it does not take any named arguments', async function() {
+      expectEmberError(new Error('Assertion Failed: The `element` helper does not take any named arguments'));
+
+      await render(hbs`
+        <div>
+          {{#let (element "h1" id="content") as |Tag|}}
+            <Tag id="content">hello world!</Tag>
+          {{/let}}
+        </div>
+      `);
+    });
+
+    test('it does not take a block', async function() {
+      expectEmberError(new Error('Assertion Failed: Helpers may not be used in the block form, for example {{#element}}{{/element}}. Please use a component, or alternatively use the helper in combination with a built-in Ember helper, for example {{#if (element)}}{{/if}}.'));
+
+      await render(hbs`
+        <div>
+          {{#element "h1"}}hello world!{{/element}}
+        </div>
+      `);
+    });
```

Comment:
@rwjblue This test is failing on old LTS due to the bounds tracking issue, (trying to teardown the block, presumably). Is there a workaround? If not I suppose I can flag the test with compatibility helpers.